tests/unit/test_network_connections.py

```diff
@@ -60,7 +60,7 @@ class Python26CompatTestCase(unittest.TestCase):
     def assertRaisesRegex(self, exception, regex, *args, **kwargs):
         if sys.version[:3] == "2.6":
             self.assertRaises(exception, *args, **kwargs)
-        elif sys.version[:3] < "3.2":
+        elif sys.version[:3] < "3.2" and sys.version[3] == ".":
```

Comment:
why not use `sys.version_info`?

`if sys.version_info[:2] < (3,2):`

Even the doc says: "Do not extract version information out of it, rather, use version_info and the functions provided by the platform module." (https://docs.python.org/3/library/sys.html).

Reply:
+1

Reply:
I updated. Thanks